lib/routes/cf/v2.js

```diff
@@ -13,9 +13,9 @@ const instanceRouter = express.Router({
 
 /* Service Broker API Router */
 router.use(middleware.basicAuth(config.username, config.password));
-if (_.includes(['production', 'test'], process.env.NODE_ENV)) {
-  router.use(controller.handler('apiVersion'));
-}
+//if (_.includes(['production', 'test'], process.env.NODE_ENV)) {
```

Comment:
why is this commented ?

Reply:
The apiVersion check was skipped unless it is a production/test system. That fails the osb checker tools. Hence, I have enabled it.